content/efcore/getting-started/samples/shared/Address.cs

```diff
@@ -0,0 +1,6 @@
+public class Address
+{
+    public int Id { get; set; }
+    public String City { get; set; }
```

Comment:
In order for the examples to work, we either need to add `using System;` or make the data type of lines 4 and 5 `string`.